Makefile.am

```diff
@@ -147,6 +149,7 @@ libmesos_tests_la_CPPFLAGS =				\
   -I$(MESOS_BUILD_DIR)/include/mesos			\
   -I$(ZOOKEEPER)/include				\
   -I$(ZOOKEEPER)/generated				\
+  -I$(NVML)						\
```

Comment:
We may actually consider using `-isystem` here instead as we do not really care for any warnings triggered by those headers.

On the other hand, we should actually consider doing these explicit includes for all inherited dependencies. There are other cases like boost which gets inherited via stout. But that is certainly not subject of this PR.